content/docs/ui/managing-contacts/email-address-validation.md

```diff
@@ -14,23 +14,23 @@ seo:
 ---
 <call-out type="warning">
 
-Email Validation is currently in beta. During beta, any participants with Pro or Premier accounts will have access to this feature. Functionality and performance is subject to change while in beta.
+Email Validation API is currently in beta. Customers with Pro or Premier Email API plans will have access to this feature free during Beta, once generally available [this pricing](https://sendgrid-email-validation.api-docs.io/v3/getting-started/authentication) will apply. Functionality and performance is subject to change while in beta.
```

Comment:
Link should be https://sendgrid.com/solutions/email-validation-api/#email-val-link ?

Reply:
updated